#Camera topic here
camera_topic: "/usb_cam/image_raw"
queue_size: 1

# Set your detection topic here
detection_topic: "/darknet_ros/bounding_boxes"

# Set your output topic here
tracker_topic: "/sort_track"

# Threshold for Costs between detections and tracks
cost_threhold: 15

# min_hits
min_hits: 1

# max age
max_age: 5

image_view:

  enable_opencv: true
  wait_key_delay: 1
  enable_console_output: true

camera_parameters:
  image_width: 640
  image_height: 480
  camera_name: camera
  translation : [0, 0, 0]
  rotation : [0, -1, 0, -1, 0, 0, 0, 0, -1]
  camera_matrix:
    rows: 3
    cols: 3
    data: [358.4701893587005, 0, 323.8761965975529, 0, 359.5166951760163, 233.6390334982792, 0, 0, 1]
  distortion_model: plumb_bob
  distortion_coefficients:
    rows: 1
    cols: 5
    data: [-0.3263238172477361, 0.08120101574385939, -0.0001620190119734289, -0.0009502949222392523, 0]
  rectification_matrix:
    rows: 3
    cols: 3
    data: [1, 0, 0, 0, 1, 0, 0, 0, 1]
  projection_matrix:
    rows: 3
    cols: 4
    data: [252.01708984375, 0, 321.790390797134, 0, 0, 294.44384765625, 230.0517886521702, 0, 0, 0, 1, 0]
